sp800-63c/sec8_security.md

```diff
@@ -5,196 +5,60 @@
-    to the request by the RP.
-
-	2.  Responses to assertion requests are bound to the corresponding
-    requests by message order, as in HTTP, provided that assertions and
-    requests are protected by a protocol such as TLS that can detect and
-    disallow malicious reordering of packets.
+Because of the additional party, the IdP, in the authentication process, there are additional opportunities for attackers to compromise federated identity transactions. This section summarizes many of the attacks and their mitigations that are applicable when federation is being used.
+
+###8.1. Federation Threats
+
+As in non-federated authentication, the motivation of attackers is typically to gain access (or a greater level of access) to a resource or service provided by an RP. Attackers may also attempt to impersonate a subscriber. Rogue or compromised IdPs, RPs, user endpoints, and parties outside of a typical assertion transaction are potential attackers. To accomplish their attack, they might intercept or modify assertions and assertion references. Furthermore, two or more entities may attempt to subvert assertion protocols by directly compromising the integrity or confidentiality of the assertion data. For the purpose of these types of threats, authorized parties who attempt to exceed their privileges are considered attackers.
+
+In some cases, the subscriber is issued some secret information so that they can be recognized by the RP. The knowledge of this information distinguishes the subscriber from attackers who wish to impersonate them. In the case of holder-of-key assertions, this secret could already have been established with the IdP prior to the initiation of the assertion protocol. In other cases, the IdP will generate a temporary secret and transmit it to the authenticated subscriber for this purpose. When this secret is used to authenticate to the RP, this temporary secret is referred to as a *secondary authenticator*. Examples of secondary authenticators include assertions in the front-channel model, session keys in Kerberos, assertion references in the back-channel model, and cookies used for session management.
+
+<div class="text-center" markdown="1">
+
+**Table 8-1 Federation Threats**
+
+</div>
+
+| **Federation Threats/Attacks**  | **Description**  | **Examples** |
+|---------------------------------|------------------|--------------|
+| Assertion manufacture or modification | The attacker generates a false assertion | Compromised IdP asserts identity of a claimant who has not properly authenticated. |
+| | The attacker modifies an existing assertion | Compromised proxy that changes AAL of an authentication assertion |
+| Assertion disclosure | Assertion visible to third party | Network monitoring reveals subscriber address of record to an outside party |
+| Assertion repudiation by the IdP | IdP later claims not to have signed transaction | Credit card transaction cannot be confirmed |
```

Comment:
It doesn't say "at the RP" in the current text, adding something like that is what I suggested above.